pyproject.toml

```diff
@@ -26,7 +26,7 @@ packages = [
 [tool.poetry.dependencies]
 python = "^3.6"
 colour = "*"
-numpy = "*"
+numpy = "1.19.3"
```

Comment:
Question: why is this necessary in this PR?

Reply:
This is an issue in Windows 10 where poetry installs version numpy "1.19.4" which doesn't fails to build and results in a error on install. This is windows specific and can be removed, but would then require care to Windows users. The error message links to this[ site](https://developercommunity.visualstudio.com/content/problem/1207405/fmod-after-an-update-to-windows-2004-is-causing-a.html): which is not a NumPy issue, but a windows issue.

Reply:
I would be strongly against it to lock over a version. Now, which version exactly doesn't create a issue? I see 1.19.4 would not create a issue. Why 1.19.3 is locked then? Also manim, don't need to do anything for this as did by other libraries which uses numpy. Please revert it.

Reply:
@jsonvillanueva could you please file a new issue to take care of this in a separate PR? And revert it to the wildcard on this PR :)

Reply:
This is on a fresh installation of Windows 10 Version 20H2. I'm guessing you're running a Version earlier than 2004/10.0.19041.488 since you're [not facing issue discussed here](https://developercommunity.visualstudio.com/comments/1243113/view.html)? Anyways, 1.19.3 is the latest version that worked across Windows, Manjaro, and OS X, this is why I changed this. If the changes are completely reverted, then it would be nice to warn users on Windows 10 ^10.0.19041.488 and provide instructions how to downgrade NumPy (e.g. ` pip install numpy==1.19.3`).

The alternative is to downgrade NumPy in the pyproject.toml for the release with `<= 1.19.3` syntax temporarily until Windows patches this in 2021 so that poetry installations works out of the box for everyone until manim-0.1.1. Users can then choose to upgrade to 1.19.4 if they really need the latest and greatest NumPy. IMO it's a better user overall experience to have everyone install without issues and have those that can upgrade if they need 1.19.4 vs. leaving out some Windows users and requiring them to fix dependencies to install properly.